lib/services/gitlab_issues.rb

```diff
@@ -0,0 +1,323 @@
+class AhaServices::GitlabIssues < AhaService
+    title 'GitLab Issues'
+    caption 'Send features to GitLab Issues'
+
+    password :private_token
+    string :server_url, description: 'If you are using your own GitLab server please enter your server URL without a trailing slash (https://example.com/api/v3). If you are using gitlab.com leave this field empty.',
+                        label: 'Server URL'
+    install_button
+    select :project, collection: -> (meta_data, _data) do
+        return [] if meta_data.nil? || meta_data.repos.nil?
+        meta_data.repos.sort_by(&:name).collect { |repo| [repo.full_name, repo.full_name] }
+    end
+
+    select :mapping, collection: [
+        ['Feature -> Issue, Requirement -> Issue', 'issue-issue'],
+        ['Feature -> Issue, Requirement -> Checklist item', 'issue-checklist']
+    ], description: 'Choose how features and requirements in Aha! will map to issues and checklists in GitLab.'
+
+    string :due_date_phase, description: 'The name of the phase to find the due date of a release', label: 'Due Date Phase'
```

Comment:
Ah got it. I think that makes sense and is a fine enhancement. I would rewrite the description as: 'The name of a release phase whose end date should be used as the milestone date in GitLab when sending releases. Falls back to the release date if not set or if the named release phase does not exist.'

I'd also add something similar to the gitlab_issues.md documentation, probably as an indented bullet after the "Aha! releases will be created as milestones in GitLab." line